docs/docs/cli/kratos-courier-watch.md

```diff
@@ -0,0 +1,40 @@
+---
+id: kratos-courier-watch
+title: kratos courier watch
+description: kratos courier watch Starts the ORY Kratos message courier
+---
+
+<!--
+This file is auto-generated.
+
+To improve this file please make your change against the appropriate "./cmd/*.go" file.
+-->
+
+## kratos courier watch
+
+Starts the ORY Kratos message courier
+
+### Synopsis
+
+Starts the ORY Kratos message courier
+
+```
+kratos courier watch [flags]
+```
+
+### Options
+
+```
+  -h, --help   help for watch
```

Comment:
hm, for some reason the cli docs aren't listing the `--expose-metrics-port` flag

Reply:
Should be updated automatically by the CI pipeline!